FrontEnd/src/components/Browse/Browse.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import { ComponentMeta, Story } from '@storybook/react';
+import { BrowsePure, BrowsePureProps } from './Browse';
+
+export default {
+  title: 'Browse',
+  component: BrowsePure,
+  argTypes: {},
+} as ComponentMeta<typeof BrowsePure>;
+
+const Template: Story<BrowsePureProps> = (args) => {
+  return (
+    <MemoryRouter initialEntries={['/Browse']}>
+      <BrowsePure {...args} />
+    </MemoryRouter>
+  );
+};
+
+export const Loaded = Template.bind({});
+Loaded.args = {
+  isLoaded: true,
+  plot: {
+    plotName: 'some plot',
+    plotNumber: 1,
+  },
+  setPlot: () => undefined,
+  plotRows: {
+    '33333847387434.csv': {
```

Comment:
Hopefully we can get some more accurate data in here sometime soon, but this works well for now!